src/Gui/Selection.cpp

```diff
@@ -2067,11 +2067,7 @@ PyObject *SelectionSingleton::sCountObjectsOfType(PyObject * /*self*/, PyObject
         return NULL;
 
     unsigned int count = Selection().countObjectsOfType(objecttype, document, resolve);
-#if PY_MAJOR_VERSION < 3

```

Comment:
This is backwards, it's the other branch that needs to be kept (the condition is the opposite of the usual >=3).

Reply:
Fixed in e26f7e42c8b80eb5ae30049ec18b2aa4d257200f